app/uk/gov/hmrc/apisubscriptionfields/config/microserviceGlobal.scala

```diff
@@ -25,16 +25,16 @@ import uk.gov.hmrc.play.microservice.filters.{AuditFilter, LoggingFilter, Micros
 
 
 object ControllerConfiguration extends ControllerConfig {
-  lazy val controllerConfigs = Play.current.configuration.underlying.as[Config]("controllers")
+  lazy val controllerConfigs: Config = Play.current.configuration.underlying.as[Config]("controllers")
```

Comment:
Is this change really needed?
Do you like to write always the type?

In Scala I specify the type only when needed.
In fact, Scala has a built-in type inference mechanism.
I always try to to write the minimum number of lines/code in Scala, unless the code is not readable.

I wonder why you propose this change.
I guess it is your personal preference.
Happy to merge, but I am curious to know why this change.

Similar comment for the same changes in this file.

Reply:
done